File: python/carBasic.py
from car import Car

class CarBasic(Car):
  brand = str
  model = str
  
  def __init__(self, license, driver, passengers, brand, model):
    super(CarBasic, self).__init__(license, driver, passengers)
    self.brand = brand
    self.model = model
    
    
    
class UberX(CarBasic):
  
  def __init__(self, license, driver, passengers, brand, model):
    super(UberX, self).__init__(license, driver, passengers, brand, model)
    

class UberPool(CarBasic):
  
  def __init__(self, license, driver, passengers, brand, model):
    super(UberPool, self).__init__(license, driver, passengers, brand, model)

File: js/CarBasic.js
class CarBasic extends Car {
  brand;
  model;

  constructor(license, driver, passenger, brand, model) {
    super(license, driver, passenger);
    this.brand = brand;
    this.model = model;
  }
}

class UberX extends CarBasic {

  constructor(license, driver, passenger, brand, model){
    super(license, driver, passenger, brand, model);
  }
}

class UberPool extends CarBasic {

  constructor(license, driver, passenger, brand, model){
    super(license, driver, passenger, brand, model);
  }
}